functional-test/src/test/java/org/zanata/feature/project/EditProjectLanguagesTest.java

```diff
@@ -129,4 +130,61 @@ public void setLanguageAliasTest() {
                 .isEqualTo("pl-PL")
                 .as("The alias was set");
     }
+
+    @Feature(summary = "The administrator can remove an alias for a project " +
+            "language",
+            tcmsTestPlanIds = 5316, tcmsTestCaseIds = 0)
+    @Test
+    @Ignore("RHBZ1199371")
```

Comment:
It would be interesting if it checked the bug status, and started running it again automatically when the bugfix is deployed.